js/controller/BoardController.js

```diff
@@ -200,6 +200,27 @@ app.controller('BoardController', function ($rootScope, $scope, $stateParams, St
 		CardService.archive(card);
 		StackService.removeCard(card);
 	};
+    $scope.isCurrentUserAssigned = function (card) {
+        if (! CardService.get(card.id).assignedUsers) {
+            return false;
+        }
+        var userList = CardService.get(card.id).assignedUsers.filter(function (obj) {
+            return obj.participant.uid === OC.getCurrentUser().uid;
+        });
+		return userList.length === 1;
+    };
+    $scope.cardAssignToMe = function (card) {
+        CardService.assignUser(card, OC.getCurrentUser().uid)
+            .then(
+                function() {StackService.updateCard(card);}
+            );
+        $('.popovermenu').addClass('hidden');
```

Comment:
We should avoid using jquery in that way. Maybe you can have a look why the popovermenu is not closed in the directive that is opening it (see js/directive/appPopoverMenuUtils.js)

Reply:
Hi,

I don't manage to make it work with appPopoverMenuUtils.js.
I believe the click event is stopped by several stopPropagation at different levels.

I don't manage to make it work without entering deeply in the application code. I believe I don't have the necessary JS skills to make it work without breaking all the app :-)

Can you help me debug that point ?

Best regards,
Philippe